pkg/reloader/reloader_test.go

```diff
@@ -114,7 +114,6 @@ config:
 	for {
 		select {
 		case <-ctx.Done():
-			break
```

Comment:
I think this is a solution to the wrong bug actually! I suspect that this case actually wanted to break out of the `for` loop rather than the `select`. To fix this, we would need to put a label on the loop, e.g `Outer:` on top of the `for` loop and then change this line to `break Outer`

Reply:
that makes total sense - accidental bliss :D Thanks!